### PR TITLE
Profile Page updated

### LINKED ISSUE #32: Profile Page updated along with new background style

### PR DESCRIPTION
**Pull request by GavinGray11**

### CODE REVIEW

File: views/index.html [hunk 1]
```diff
@@ -17,56 +17,182 @@
     <!-- GOOGLE FONTS -->
     <link rel="preconnect" href="https://fonts.gstatic.com">
     <link href="https://fonts.googleapis.com/css2?family=Fredoka+One&display=swap" rel="stylesheet">
+
 </head>
 
 <body>
 
     <nav class="navbar navbar-expand-lg navbar-dark">
-        <a class="navbar-brand" href="/"><i class="fas fa-paw"></i></a>
```
Comment (sudo-update) [inline views/index.html:25]: will this remove the paw print button's functionality?

File: views/profile.html [hunk 2]
```diff
@@ -1,101 +1,209 @@
+<!DOCTYPE html>
 <html>
 
 <head>
-  <title> Profile - Dog Site </title>
```
Comment (sudo-update) [inline views/profile.html:4]: This might need to stay, we can change the actual string's value though

File: views/profile.html [hunk 3]
```diff
@@ -1,101 +1,209 @@
...
+  <!-- GOOGLE FONTS -->
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Fredoka+One&display=swap" rel="stylesheet">
+
 </head>
+
 <body>
 
   <nav class="navbar navbar-expand-lg navbar-dark">
-    <a class="navbar-brand" href="/"><i class="fas fa-paw"></i></a>
```
Comment (sudo-update) [inline views/profile.html:22]: paw print button, might need to stay as "/"

File: views/profile.html [hunk 4]
```diff
@@ -1,101 +1,209 @@
...
+  </section>
+  <!-- playdates section end -->
+  <section class="profile-info">
+    <div class="profile-circle border-circle">
+      <img src="/Ollie.png" class="avatar img-raised rounded-circle img-fluid" alt="profilepic">
+      <div class="container text-center user-content">
+        <div class="row">
+
+          <div class="name">
+            <h3 class="">Christian Louboutin</h3>
```
Comment (sudo-update) [inline views/profile.html:184]: I would change this name to yours

File: views/profile.html [hunk 5]
```diff
@@ -128,20 +236,22 @@ <h4 class="m-t-10 m-b-5">Sean J</h4>
                     <!-- begin timeline-body -->
                     <div class="timeline-body">
                       <div class="timeline-header">
-                        <span class="userimage"><img src="https://bootdey.com/img/Content/avatar/avatar3.png" alt=""></span>
-                        <span class="username"><a href="javascript:;">Sean J</a> <small></small></span>
-                        <span class="pull-right text-muted">6 Views</span>
+                        <span class="userimage"><img src="https://bootdey.com/img/Content/avatar/avatar3.png"
+                            alt=""></span>
+                        <span class="username"><a href="javascript:;">Sean Ngu</a> <small></small></span>
```
Comment (sudo-update) [inline views/profile.html:241]: Sean J

File: views/profile.html [hunk 6]
```diff
@@ -128,20 +236,22 @@ <h4 class="m-t-10 m-b-5">Sean J</h4>
...
+                        <span class="pull-right text-muted">18 Views</span>
                       </div>
                       <div class="timeline-content">
                         <p>
-                          I look terrible.  But only for now.  Sean is hard at work learning how to web develop.  Soon, I will
```
Comment (sudo-update) [inline views/profile.html:137]: some more personalized text vs template txt, not very important though

File: views/profile.html [hunk 7]
```diff
@@ -191,17 +304,21 @@ <h4 class="m-t-10 m-b-5">Sean J</h4>
                     <!-- begin timeline-body -->
                     <div class="timeline-body">
                       <div class="timeline-header">
-                        <span class="userimage"><img src="https://bootdey.com/img/Content/avatar/avatar3.png" alt=""></span>
-                        <span class="username">Sean J</span>
+                        <span class="userimage"><img src="https://bootdey.com/img/Content/avatar/avatar3.png"
+                            alt=""></span>
+                        <span class="username">Sean Ngu</span>
```
Comment (sudo-update) [inline views/profile.html:309]: name

File: views/profile.html [hunk 8]
```diff
@@ -221,29 +338,35 @@ <h4 class="m-t-10 m-b-5">Sean J</h4>
...
```
Comment (sudo-update) [inline views/profile.html:343]: name

File: views/profile.html [hunk 9]
```diff
@@ -221,29 +338,35 @@ <h4 class="m-t-10 m-b-5">Sean J</h4>
...
                         <span class="pull-right text-muted">1,282 Views</span>
                       </div>
                       <div class="timeline-content">
                         <p class="lead">
                           <i class="fa fa-quote-left fa-fw pull-left"></i>
-                          2014 was a crazy year.  During this year, I became a club promoter in San Francisco.  I was coming out of my shell and becoming myself.
+                          Quisque sed varius nisl. Nulla facilisi. Phasellus consequat sapien sit amet nibh molestie
```
Comment (sudo-update) [inline views/profile.html:349]: personalized text

File: views/profile.html [hunk 10]
```diff
@@ -255,24 +378,29 @@ <h4 class="m-t-10 m-b-5">Sean J</h4>
                     <!-- begin timeline-body -->
                     <div class="timeline-body">
                       <div class="timeline-header">
-                        <span class="userimage"><img src="https://bootdey.com/img/Content/avatar/avatar3.png" alt=""></span>
-                        <span class="username">Sean J</span>
+                        <span class="userimage"><img src="https://bootdey.com/img/Content/avatar/avatar3.png"
+                            alt=""></span>
+                        <span class="username">Sean Ngu</span>
```
Comment (sudo-update) [inline views/profile.html:383]: name

File: views/profile.html [hunk 11]
```diff
@@ -255,24 +378,29 @@ <h4 class="m-t-10 m-b-5">Sean J</h4>
...
                       </div>
                       <div class="timeline-content">
                         <h4 class="template-title">
                           <i class="fa fa-map-marker text-danger fa-fw"></i>
-                          715 Harrison St, San Francisco, CA 94107
+                          795 Folsom Ave, Suite 600 San Francisco, CA 94107
                         </h4>
-                        <p>The famous club itself.  Made many memories driving into the city with my best friend on a Friday night after a long week of college studies.</p>
+                        <p>In hac habitasse platea dictumst. Pellentesque bibendum id sem nec faucibus. Maecenas
+                          molestie, augue vel accumsan rutrum, massa mi rutrum odio, id luctus mauris nibh ut leo.</p>
```
Comment (sudo-update) [inline views/profile.html:392]: personalized

**Review by sudo-update — COMMENTED** [review]
just a couple minor changes, and a couple changes I think were unintentional (the paw print button)

File: views/index.html (hunk 1 above)
Comment (GavinGray11) [reply views/index.html:25]: yes, I'll fix it right now.

**Pull request merged**